doc/Media2.xml

```diff
@@ -2167,6 +2167,244 @@
+        <variablelist role="op">
+          <varlistentry>
+            <term>request</term>
+            <listitem>
+			<para role="param">Token - optional[tt:ReferenceToken]</para>
+              <para role="text"> Optional token associated with the audio clip.</para>
+              <para role="param">Configuration - 
+                [tr2:AudioClip]</para>
+              <para role="text"> Audio clip configuration to add.</para>
+            </listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>response</term>
+            <listitem>
+              <para role="param">Token - [tt:ReferenceToken]</para>
+              <para role="text">Unique token of the audio clip to be uploaded.</para>
+              <para role="param">UploadUri - [xs:anyURI]</para>
+              <para role="text">A URL to which the audio clip can be uploaded.</para>
+              <para role="param">ExpiryTime - [xsd:DateTime]</para>
+              <para role="text">Expiry time by which the client should upload the audio clip to the device.</para>             
+            </listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>faults</term>
+            <listitem>
+              <para role="param">env:Receiver - ter:Action - ter:MaxAudioClipLimit</para>
+              <para role="text">The maximum number of audio clip configurations supported by the device has been reached.</para>
+              <para role="param">env:Sender - ter:InvalidArgVal - ter:InvalidConfig</para>
+              <para role="text">The configuration parameters are not possible to set.</para>         
+            </listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>access class</term>
+            <listitem>
+              <para role="access">WRITE_SYSTEM</para>
+            </listitem>
+          </varlistentry>
+        </variablelist>
+        <para></para>
+        <para><b>Note:</b> Audio clip uploads to the device can fail in the following scenarios, and a specific HTTP error code should be returned to the client when an upload fails.</para>
+        <itemizedlist>
+          <listitem>
+            <para>When the upload URL has expired, the device should return a HTTP 410 error to the client.</para>
+          </listitem>
+          <listitem>
+            <para>When the format of the audio clip upload does not match the Type parameter in the audio clip configuration, the device should return a HTTP 415 Unsupported Media Type error to the client.</para>
+          </listitem>
+          <listitem>
+            <para>When the size of the audio clip upload exceeds the MaxAudioClipSize parameter in KB, the device should return a HTTP 413, Request Entity Too Large error to the client.</para>
+          </listitem>
+        </itemizedlist>              
+      </section>    
+	  <section xml:id="section_wvd_dzg_rye">
+        <title>SetAudioClip</title>
```

Comment:
Is it possible to modify all clip parameters?

Reply:
The format of the audio clip cannot be changed during the SetAudioClip operation.